From 58ffa1b413690dbfdea86c068510339fe1573c33 Mon Sep 17 00:00:00 2001
Message-Id: <58ffa1b413690dbfdea86c068510339fe1573c33.1650877238.git.geliang.tang@suse.com>
From: Christoph Hellwig <hch@lst.de>
Date: Fri, 19 Nov 2021 17:32:11 +0100
Subject: [PATCH] x86, bpf: Cleanup the top of file header in bpf_jit_comp.c
Git-commit: 58ffa1b413690dbfdea86c068510339fe1573c33
Patch-mainline: v5.17-rc1
References: bsn#207
Modified-by-SEL: No


Don't bother mentioning the file name as it is implied, and remove the
reference to internal BPF.

Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Alexei Starovoitov <ast@kernel.org>
Acked-by: Song Liu <songliubraving@fb.com>
Link: https://lore.kernel.org/bpf/20211119163215.971383-2-hch@lst.de
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 arch/x86/net/bpf_jit_comp.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/x86/net/bpf_jit_comp.c b/arch/x86/net/bpf_jit_comp.c
index 631847907786..1d7b0c69b644 100644
--- a/arch/x86/net/bpf_jit_comp.c
+++ b/arch/x86/net/bpf_jit_comp.c
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: GPL-2.0-only
 /*
- * bpf_jit_comp.c: BPF JIT compiler
+ * BPF JIT compiler
  *
  * Copyright (C) 2011-2013 Eric Dumazet (eric.dumazet@gmail.com)
- * Internal BPF Copyright (c) 2011-2014 PLUMgrid, http://plumgrid.com
+ * Copyright (c) 2011-2014 PLUMgrid, http://plumgrid.com
  */
 #include <linux/netdevice.h>
 #include <linux/filter.h>
-- 
2.34.1

